src/main/java/com/revature/models/User.java

```diff
@@ -22,7 +22,7 @@
 
 @Entity
 @JsonIgnoreProperties({ "hibernateLazyInitializer", "handler" })
-@Table(name = "sms_users", schema = "user_service")
```

Comment:
We should leave the schema name alone. This could break the deployed environment.

Reply:
ok I'm going to revert those changes to the way they were before. It was only for local testing.